auto_submit/lib/service/validation_service.dart

```diff
@@ -146,9 +146,26 @@ class ValidationService {
       }
     }
     // If we got to this point it means we are ready to submit the PR.
-    bool processed = await processMerge(config, result, messagePullRequest);
-    if (processed) await pubsub.acknowledge('auto-submit-queue-sub', ackId);
-    log.info('Ack the processed message : $ackId.');
+    try {
+      bool processed = await processMerge(config, result, messagePullRequest);
+      if (processed) {
+        await pubsub.acknowledge('auto-submit-queue-sub', ackId);
```

Comment:
We should ack the message for any case till this point.

Reply:
So even when we temporarily ignore in the result action we need to ack the message? Because that is the only case we are in this path when processing the pull request.

Reply:
For the case of `ignore`, we should keep.
For for any place where we remove the label, we should ack.

Reply:
Okay. I added the acknowledge everywhere we remove the label. It seems that in any case but ignore we then remove the label.